src/style.css

```diff
@@ -1,3 +1,102 @@
-body {
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap';
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  --link-color: #00ACDC;
+}
+
+body {
+ font-family: Roboto, sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.logo__link {
+  position: fixed;
+  left: 50;
+  top: 10;
+  width: 40px;
+  height: 40px;
+}
+
+nav{
```

Comment:
Just a small reminder to always put a space between the property and {

(no big deal, we all forget this sometimes and it'll not impact your code)